src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -415,16 +415,18 @@ public function sendContent(): static
     /**
      * Sends HTTP headers and content.
      *
+     * @param bool $finishRequest Whether fastcgi_finish_request() and alike functions should be called or not
+     *
      * @return $this
      */
-    public function send(): static
+    public function send(/* bool $finishRequest = true */): static
     {
         $this->sendHeaders();
         $this->sendContent();
 
-        if (\function_exists('fastcgi_finish_request')) {
+        if (\function_exists('fastcgi_finish_request') && (1 <= \func_num_args() ? func_get_arg(0) : true)) {
```

Comment:
The `$finishRequest` variable should be created on a separate line. This way, the diff with 7.0 will be minimal.

Reply:
yes, this is overkill and makes maintenance harder